BENCH

```diff
@@ -0,0 +1,11 @@
+FROM shinsaku0523/zkevm:latest
```

Comment:
Why does this contain your username? Not sure I follow

Reply:
I only access my docker registry...
Do we have docker registry?